app/kumactl/cmd/install/testdata/install-control-plane.defaults.golden.yaml

```diff
@@ -2046,11 +2046,116 @@ spec:
+                                of live destination dataplane proxies below which
+                                load balancing to the next priority starts. Example:
+                                If you configure failoverThreshold to 70, and you
+                                have deployed 10 destination dataplane proxies. Load
+                                balancing to next priority will start when number
+                                of live destination dataplane proxies drops below
+                                7. Default 50'
+                              properties:
+                                percentage:
+                                  anyOf:
+                                  - type: integer
+                                  - type: string
+                                  x-kubernetes-int-or-string: true
+                              required:
+                              - percentage
+                              type: object
+                            localZone:
+                              description: LocalZone defines locality aware load balancing
+                                priorities between dataplane proxies inside a zone
+                              properties:
+                                affinityTags:
+                                  description: AffinityTags list of tags for local
+                                    zone load balancing.
+                                  items:
+                                    properties:
+                                      key:
+                                        description: Key defines tag for which affinity
+                                          is configured
+                                        type: string
+                                      weight:
+                                        description: 'Weight of the tag used for load
+                                          balancing. The bigger the weight the bigger
+                                          the priority. Percentage of local traffic
+                                          load balanced to tag is computed by dividing
+                                          weight by sum of weights from all tags.
+                                          For example with two affinity tags first
+                                          with weight 80 and second with weight 20,
+                                          then 80% of traffic will be redirected to
+                                          the first tag, and 20% of traffic will be
+                                          redirected to second one. Setting weights
+                                          is not mandatory. When weights are not set
+                                          control plane will compute default weight
+                                          based on list order. Default: If you don''t
```

Comment:
nitpick:

```suggestion
                                          based on list order. Default: If you don't
```

Reply:
These are autogenerated, if they render correctly wherever we copy them to the website then I'm fine with that. Just something to check after you merge I guess. Or maybe someone else from the team knows if they render correctly.